app/models/participant_declaration/npq.rb

```diff
@@ -7,6 +7,8 @@ class ParticipantDeclaration::NPQ < ParticipantDeclaration
 
   has_many :statements, class_name: "Finance::Statement::NPQ", through: :statement_line_items
 
+  has_many :outcomes, class_name: "ParticipantDeclaration::Outcome::NPQ", foreign_key: "participant_declaration_id"
```

Comment:
In terms of what outcome means, I think it's related to a participant as a whole passing rather than to a declaration which is just to mark that the participant has completed the course. So maybe our model should be named "ParticipantOutcome::NPQ"? Same with renaming the table?

Reply:
Renamed https://github.com/DFE-Digital/early-careers-framework/pull/2755/commits/92d5dcb8c90931ec2fa37b69128569941b25a60c
Renamed table https://github.com/DFE-Digital/early-careers-framework/pull/2755/commits/5f08a9843854bd6dde4aed43d52a3466d20281ad